src/crypto/backup.ts

```diff
@@ -111,6 +112,7 @@ export class BackupManager {
     public backupInfo: IKeyBackupInfo | undefined; // The info dict from /room_keys/version
     public checkedForBackup: boolean; // Have we checked the server for a backup we can use?
     private sendingBackups: boolean; // Are we currently sending backups?
+    private sessionLastCheckAttemptedTime: Record<string, number> = {}; // When did we last try to check the server for a given session id?
     constructor(private readonly baseApis: MatrixClient, public readonly getKey: GetKey) {
```

Comment:
blank line to appease future linter:
```suggestion

    constructor(private readonly baseApis: MatrixClient, public readonly getKey: GetKey) {
```